docs/csharp/language-reference/builtin-types/reference-types.md

```diff
@@ -80,7 +80,61 @@ for (int i = 0; i < str.Length; i++)
 // Output: t e s t
 ```
 
-String literals are of type `string` and can be written in two forms, quoted and `@`-quoted. Quoted string literals are enclosed in double quotation marks ("):
+### String literals
+
+String literals are of type `string` and can be written in three forms, raw, quoted, and `@`-quoted. Raw string literals are enclosed in a minimum of three double quotation marks ("""):
+
+```csharp
+"""
+This is a multi-line
+    string literal with the second line indented.
+"""
+```
+
+*Raw string literals* are available beginning in C# 11. Raw string literals can contain arbitrary text without requiring escape sequences. Raw string literals can include whitespace and new lines, embedded quotes, and other special characters. You can even include a sequence of three (or more) double quote characters. If your text requires an embedded sequence of quotes, you start and end the raw string literal with more quote marks, as needed:
+
+```csharp
+"""""
+This raw string literal has four """", count them: """" Four! 
+embedded quote characters in a sequence. That's why it starts and ends
+with five double quotes.
+
+You could extend this example with as many embedded quotes as needed for your text.
+"""""
+```
+
+Raw string literals typically have the starting and ending quote sequences on separate lines from the embedded text. Multiline raw string literals supports strings that are themselves quoted strings:
```

Comment:
this is good to say, but we should also add (if not already there) that the newline on the opening line, and the newline of the last content line are not included in the final content.